build/circuitbreaker.yml

```diff
@@ -2,6 +2,7 @@ pr:
   branches:
     include:
     - main
+    - release/3.*
```

Comment:
This could be `release/*`, but perhaps we should leave out these filters entirely in 4.0? I'm asking because I've found that a PR based off another PR-branch does not run cibuild, until the first PR is merged so the second PR gets retargeted to main. If we can't leave out the branches/include to retain path-based exclusions, we can try just including *.

Reply:
I think the original main intent was to exclude `release/2.*` based on the expectation (or knowledge?) that the pipelines wouldn't run there, but it's been long enough that I don't recall for sure. Regardless, if we're going to change it to `*` that's probably better done on `main` than this release branch